src/Temporalio/Bridge/Interop/Interop.cs

```diff
@@ -141,6 +141,18 @@ internal partial struct ClientKeepAliveOptions
         public ulong timeout_millis;
     }
 
+    internal partial struct ClientHttpConnectProxyConfig
+    {
+        [NativeTypeName("struct ByteArrayRef")]
+        public ByteArrayRef target_addr;
```

Comment:
```suggestion
        public ByteArrayRef target_host;
```

Reply:
Unresolving. This doesn't seem to match the Rust name.

Reply:
Sorry, misunderstood you. I thought you were veto'ing my proposed name change comment above.

Reply:
Updated.